content/docs/scaffolder/manipulate-json-files/index.md

```diff
@@ -0,0 +1,191 @@
+    - id: log-message
+      name: Log Message
+      action: debug:log
+      input:
+        message: Template execution started
+
+    - id: create-file
+      name: Create file
+      action: roadiehq:utils:fs:write
+      input:
+        path: ./template.json
+        content: >
+          {
+            "existsing": "content",
+            "nested": {
+              "one" :{
+                "kinda": "deep"
+                }
+            }
+          }
+
+    - id: merge
+      name: Merge
+      action: roadiehq:utils:merge
+      input:
+        path: ./template.json
+        content: >
+          {"foo": "bar"}
+
+    - id: parse
+      name: read file
+      action: roadiehq:utils:fs:parse
+      input:
+        path: ./template.json
+        parser: json
+
+    - id: serialize
+      name: Serialize
+      action: roadiehq:utils:serialize:json
+      input:
+        data: ${{ steps['parse'].output.content }}
+
+    - id: log
+      name: Console.log
+      action: debug:log
+      input:
+        message: |
+          Content: `${{ steps['serialize'].output.serialized }}`
+```
+
+## Breakdown
+
+### Parameters
+
+We didnt use any parameters for this demonstration as we don't need any user input to showcase the power of these actions.
+
+### Steps
+
+#### debug:log
+
+Uses the debug:log action to output a log message to the backstage scaffolder UI. The message here is a general hard-coded message that indicates that the template execution has started
+
+```yaml
+- id: log-message
+  name: Log Message
+  action: debug:log
+  input:
+    message: Template execution started
+```
+
+#### roadiehq:utils:fs:write
+
+```yaml
+- id: create-file
+  name: Create file
+  action: roadiehq:utils:fs:write
+  input:
+    path: ./template.json
+    content: >
+      {
+        "existsing": "content",
+        "nested": {
+          "one": {
+            "kinda: "deep"
+          }
+        }
+      }
+```
+
+We use this action to create a test json file into the root of the workspace with the content:
+
+```json
+{
+  "existsing": "content",
+  "nested": {
+    "one": {
+      "kinda": "deep"
+    }
+  }
+}
+```
+
+#### roadiehq:utils:merge
+
+```yaml
+- id: merge
+  name: Merge
+  action: roadiehq:utils:merge
+  input:
+    path: ./template.json
+    content: >
+      {"foo": "bar"}
+```
+
+This action will merge the `input.content` into the file you specify in the `input.path` property. After the merge our `template.json` file will look like this:
+
+```json
+{
+  "existsing": "content",
+  "nested": { "one": { "kinda": "deep" } },
+  "foo": "bar"
+}
+```
+
+#### roadiehq:utils:fs:parse
+
+```yaml
+- id: parse
+  name: read file
+  action: roadiehq:utils:fs:parse
+  input:
+    path: ./template.json
+    parser: json
+```
+
+We use this action to parse the file specified at `input.path` because we know it is a json file we can use the `input.parser` with the value `json`. This will make the content of the file to be parsed into an object properly
```

Comment:
```suggestion
We use this action to parse the file specified at `input.path` because we know it is a JSON file we can use the `input.parser` with the value `json`. This will make the content of the file to be parsed into an object properly.
```